### DISCUSSION TITLE
src-placeholder?

### DISCUSSION THREAD

**Discussion by madcar** [uicrooks/shopify-foundation-theme#55] [General]
There is a src-placeholder attribute on images in various places in the theme.
What is this for? I don't see any code that uses it and I don't think it's a standard attribute.
Is it meant to be used with a lazy loading library?
BTW, thank you for making this available, it's a very helpful starting point!

**Comment by sergejcodes** [comment]
Hi!

There was previously a lazy-loading lib included which used the placeholder attribute. It's obsolete now, you can safely remove it.

**Reply by madcar** [reply]
Great thanks!
